src/vmm/src/config/mod.rs

```diff
@@ -11,10 +11,14 @@ use arg_parser::CfgArgParser;
 
 use super::{DEFAULT_HIGH_RAM_START, DEFAULT_KERNEL_CMDLINE};
 
+use builder::Builder;
+
 mod arg_parser;
 
+mod builder;
```

Comment:
Hmm, unless I'm missing something here, to be able to see the `builder` in the generated documentation, I think this should be `pub mod` (or `pub use builder::Builder` above for just the `Builder` functionality).